docs/jwst/tweakreg/README.rst

```diff
@@ -100,8 +100,9 @@ If the step parameter ``abs_refcat`` is set to 'GAIADR3', 'GAIADR2' or 'GAIADR1'
 an astrometric reference catalog then gets generated by querying
 a GAIA-based astrometric catalog web service for all astrometrically
 measured sources in the combined field-of-view of the set of input
-images. This catalog gets generated from the catalogs available
-through the GSSS STScI web services at `GSSSCATALOGS`_.
+images. This catalog is generated from the catalogs available
+through the GSSS STScI web services at `GSSSCATALOGS`_ and accounts for proper 
+motion given an epoch; defaults to 2016.0, or no proper motion correction.
```

Comment:
```suggestion
motion to a given epoch. The default epoch is 2016.0, which results in no proper motion correction.
```

Reply:
Hmmm, OK, now that I've read through the rest of the code updates (below) I see that epoch is not a user-level param, but rather gets computed from the meta data of the image being processed. So this description in the docs should be expanded to indicate that's what happens, i.e. the reference epoch is automatically derived from meta data of the image and hence the user doesn't need to know about the default 2016 value.